docs/AUTHENTICATION.md

```diff
@@ -0,0 +1,11 @@
+Authentication
+===
+
+1. frontend: Sends user to the lock interface that's provided by auth0 when required.
```

Comment:
What is a 'lock interface'? I googled for `"lock interface" auth0`, but didn't find anything that would explain to me what this is.